File: ParadigmasdaProgramacao/atividadeJava/src/Questoes/AumentoSalario.java
package Questoes;

public class AumentoSalario {

        private String nome;
        private double salario;
        private double imposto;
        private double aumento;


        public String getNome() {
            return nome;
        }

        public AumentoSalario(String nome, double salario, double imposto) {
            this.nome = nome;
            this.salario = salario;
            this.imposto = imposto;
        }

        public double salarioLiquido(double salario, double imposto) {
            this.salario -= this.imposto;

            return this.salario;
        }

        public double aumento(double salario, double aumento) {
            this.aumento = aumento;
            this.salario = this.salario + this.salario * (this.aumento/100);

            return this.salario;
        }
}

File: ParadigmasdaProgramacao/atividadeCsharp/atividadeCsharp/AumentoSalario.cs
using System;

namespace atividadeCsharp
{
    public class AumentoSalario
    {
        public AumentoSalario()
        {
        }
        private string nomeTrabalhador;
        private float salario;
        private float imposto;
        private float aumento;

        public string NomeTrabalhador { get => nomeTrabalhador; set => nomeTrabalhador = value; }
        public float Salario { get => salario; set => salario = value; }
        public float Imposto { get => imposto; set => imposto = value; }
        public float Aumento { get => aumento; set => aumento = value; }

        public float salarioLiquido(float salario, float imposto)
        {
            this.salario = salario;
            this.imposto = imposto;
            this.salario -= this.imposto;
            return this.salario;
        }

        public float novoSalario(float salario, float aumento)
        {
            this.salario = salario;
            this.aumento = aumento;
            this.salario = this.salario + this.salario * (this.aumento / 100);
            return this.salario;
        }
    }
}